src/main/java/quentin/core/Quentin.java

```diff
@@ -31,7 +30,7 @@ protected Quentin(int boardSize, InputHandlerImplementation inputHandler, Output
         this(boardSize, inputHandler, outputHandler, "player1", "player2");
     }
 
-    protected void makeMove(Stone color, Position position) throws Exception {
```

Comment:
I think we should leave exception, why specify the specific type of exceptions?

Reply:
Because should be avoided to throw Exception; It's, instead, a good practice to be more specific as possible and declare in throws statement which exceptions will be thrown (if It is necessary of course).

Reply:
Where did you read it's a good practice? However, my doubt is that if you want to expand the app and add new features and maybe new exceptions, this might become a long throws list. In the end `Exception` is the superclass of all exceptions, thus you can throw `Exception` and catch a specific exception (the catch is able to look for the actual type of the exception thrown).

Reply:
We could wrap the exceptions in logical groups in case.

Reply:
> Finished implementing the replay feature in the Console version; Could anyone implement it in the GUI version? Also, please reply to the previous comments.

I'm quite busy right now, but if nobody wants to propose itself I may do It (when I have time to do).